yapapi/log.py

```diff
@@ -101,6 +101,7 @@ def enable_default_logger(
     events.WorkerFinished: "Worker finished",
     events.DownloadStarted: "Download started",
     events.DownloadFinished: "Download finished",
+    events.UserLogMessage: "Message logged by user",
```

Comment:
Hmm, maybe `CustomMessage`, without referring to "user"?

Reply:
UserLogMessage is not perfect, but it's clear, that it's coming from the library user, not from the library itself and not from the providers. But I'm not sure.

Reply:
I get your point, I was only not sure if the term "user" is clear here.